### ISSUE TITLE
Incompatible with Latest Transmit

### ISSUE THREAD

**Issue by TechRemarker** [ramiroaraujo/alfred-transmit-workflow#5]
Have been beta testing Transmit 5 for sometime now and missed the Alfred integration since this plugin does not support Transmit 5. With it officially available to the public today wanted to make an official bug report here now for adding compatibility for Transmit 5 as I imagine there will be a rush of users downloading today.

**Comment by jaywilliams** [comment]
Yes, this is the first thing I noticed after installing the new version.

**Comment by lumpysimon** [comment]
This is one of my most used workflows, would be great if it could be updated to work with Transmit 5.

**Comment by ramiroaraujo** [comment]
:) I'll be working on this. Thanks!

**Comment by eclectic-coding** [comment]
Any updates on the Transmit 5 version?

**Comment by eduwass** [comment]
Hi @ramiroaraujo 

I suppose you haven't worked on this?

I've noticed that on Transmit 4 the favorites were on `~/Library/Application Support/Transmit/Favorites/Favorites.xml`

Did this change in Transmit 5? Is that why you dropped support? Anything I can do to help out?

Cheers,
